config/katello.migrations/200605154320-dont-use-pulpcore-rpm-on-upgrades.rb

```diff
@@ -0,0 +1,9 @@
+PUPPET_CLASS = 'katello'.freeze
+PARAM = 'use_pulp_2_for_yum'.freeze
```

Comment:
I find generic constants make the code hard to read, as I have to do a lot of mental translating to review. I'm OK with constants but prefer, for understandability, they just be a reflection of the thing they represent, e.g.

```
KATELLO = 'katello'.freeze
USE_PULP_2_FOR_YUM = 'use_pulp_2_for_yum'.freeze
```